### ISSUE TITLE
Add an option to only search for 100% nice numbers

### ISSUE THREAD

**Issue by wasabipesto** [wasabipesto/nice-rust#3]
- allows for numerous performance improvements, could probably double or triple performance
- may be able to run in parallel with normal (detailed) searches (requires database re-architecture)
- potential for abuse - no mana payout for nice-only searches

**Comment by wasabipesto** [comment]
Closing per #10 and opening a new issue for residual class filtering since it's quickly growing into a larger thing
